### ISSUE TITLE
Dev: Make updates to domain invitation emails

### ISSUE THREAD

**Issue by Katherine-Osos** [cisagov/manage.get.gov#2960]
### Story

As a recipient of a domain invitation
I want the system to recognize whether I'm invited to multiple domains and if I'm already a user of the registrar
so that I'm only sent relevant information.

### Acceptance Criteria
Make the following updates to the domain invitation email (applies to org model AND non-org model):
- [x] Display dynamic text based on whether someone was invited to a single domain or multiple domains.
- [x] In the greeting, display the user's name if they're already a "user" in our system.  
  - [x] Simply display "Hi." if not a user in the system.
- [x] Display the section about needing a Login.gov account only if the invitee is not a “user” in our system yet.
- [x] Ensure the email reflects [this updated content](https://docs.google.com/document/d/18eQRA5QWu5Ct6ICK2-FCZJf9kbbk4LklekRRfb6g9tQ/edit?tab=t.0#heading=h.gjdgxs)

When invitations are sent for domains that are part of Org Model, double check that this AC from previous ticket #3019 is still intact
- [x]  If the invitee is not part of the portfolio yet, send the [portfolio invitation email ](https://github.com/cisagov/manage.get.gov/issues/3019)AND the domain invitation email.

- [x] The above ACs apply to domain invitations send from the following:
   - [x] Django admin
   - [x]  Domain Managers page (on a domain page)
   
  
- [x] On django admin, if a new domain invitation is created then send the same emails mentioned above (domain invitation and portfolio invitation if applicable) (the alert warning will be added to this table in #2370 )

Error handling for non-unique user email:

- [ ] ~~When making a domain invitation for an existing user, check for non-unique user email and trigger error msg/log on DJA and on the form View. Send email, do not proceed with creating the user domain role.~~ @dave-kennedy-ecs @abroddrick @rachidatecs discussed handling this edge case on the oidc side

Portfolio invites on existing user:

- [x] If someone is invited to a portfolio and they are already a registrar user, we should automatically create their user portfolio permission record (auto-retrieve the portfolio invite). This applies to straight portfolio invites, also applies to the 'side-effect' portfolio invites that get triggered on domain invites.

### Additional Context
this will be added to the members page in #3295 , you can do these two in the same PR if desirable, it was just seperated in case someone wants to do them in separate PRs (and since it is a different area in the code/unit tests)

### Issue Links

- ⛔  Blocked by #3019 
-  Blocks #3295 (or do together)
- bundled in #3296
- blocks #2370 (or at least keep this one in mind while doing 2370

**Comment by abroddrick** [comment]
Refinement as we will need to discuss in the org model meeting about the impacts with us adding requests to production orgs
